### ISSUE TITLE
elastic ip binding not working

### ISSUE THREAD

**Issue by mattenat** [solutious/rudy#40]
It appears to be trying to bind the ip address too early.  This was working as recently as a couple of weeks ago.

Output from the cli:

 Executing routine: startup 
authorize port 22 access for: XXX.XXX.XXX.XXX
Associating YYY.YYY.YYY.YYY to i-6b146d03
  Error: The instance 'i-6b146d03' is not in the 'running' state
Created: m-us-east-1b-stage-app-01; i-6b146d03; 
Waiting for m-us-east-1b-stage-app-01 to boot........................
Waiting for port 22 on m-us-east-1b-stage-app-01 ........

Output from the config machines:

```
# MACHINES
--- 
:"us-east-1": 
  :ami: ami-ccf615a5
:hostname: :default
:stage: 
  :app: 
    :addresses: YYY.YYY.YYY.YYY
:prod: 
  :app: 
    :addresses: ZZZ.ZZZ.ZZZ.ZZZ
```

**Comment by delano** [comment]
Interesting! It used to be possible to assign IP addresses before a machine was in the running state. It looks like that's no longer the case. It also looks like it takes a lot longer for the instance's DNS to refresh with the new address (it was taking over 60 seconds while I was testing it just now). 

The issue is fixed in 0.9.5.004 (available in the repo and gemcutter).

**Comment by mattenat** [comment]
Thanks a bunch!
